### PR TITLE
[frontend] fix status list limit in manipulate knowledge (#6581)

### LINKED ISSUE #6581: [Playbook] The drop-down menu doesn't show everything
## Description

The drop-down menu in the "manipulate knowledge" component does not display all available statuses, even when using the search function.

## Environment

OCTI 6.0.8

## Reproducible Steps

Before making the repro case, you need to have several status set up on your platform. For example, set 3 - 4 statuses on 5 different entities (otherwise, go directly to the testing platform).

Steps to create the smallest reproducible scenario:
1. Create a playbook and add a "manipulate knowledge" component
2. Try to replace a status -> you won't find all the statuses. Neither by scrolling nor by searching

## Screenshots

Screenshot 2024-04-04 184323

### PR DESCRIPTION
**Pull request by marieflorescontact**
### Proposed changes

* Set the query limit to 100 instead of 10

### Related issues

* [issue/6581](https://github.com/OpenCTI-Platform/opencti/issues/6581)

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ class StatusField extends Component {
 
   searchStatuses() {
     fetchQuery(statusFieldStatusesSearchQuery, {
-      first: 10,
+      first: 100,
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx:101]: Reading the original issue, I'm not sure it's the source of the problem. It's not related to the status being _used_ to be queriable?

Comment (marieflorescontact) [reply opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx:101]: we want to list all entity workflow statuses, whether in use or not. which is the case. in `ToolBar.jsx` we have a similar use of the `statusFieldStatusesSearchQuery`

Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx:101]: tested with the playbook given in the issue, that do not fix the problem. IMO it's more about what the query does in the backend side

Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx:101]: after discussions: yes it fixes the issue but it seems there is an other issue more UI related, cf thread of the issue.

Comment (marieflorescontact) [reply opencti-platform/opencti-front/src/private/components/common/form/StatusField.jsx:101]: after discussion with @Jipegien , this fixes the bug but the UI needs to be improved by grouping status by entity type, which was done by @frapuks in this [pr](https://github.com/OpenCTI-Platform/opencti/pull/6707)

**Review by lndrtrbn — APPROVED** [review]

**Pull request merged**